addons/binding/org.openhab.binding.sony/README.md

```diff
@@ -0,0 +1,207 @@
+  
+## Full Example
+
+demo.Things:
+
+```
+sony:bravia:home [ ipAddress="192.168.1.72", netInterface="eth0", ping=30, retryPolling=10 ]
+```
+
+demo.items:
+
+```
+Number Bravia_IR "IR [%s]" { channel="sony:bravia:home:ir" }
+Switch Bravia_Power "Power [%s]" { channel="sony:bravia:home:power" }
+Dimmer Bravia_Volume "Volume [%s]" { channel="sony:bravia:home:volume" }
+Switch Bravia_AudioMute "Audio Mute [%s]" { channel="sony:bravia:home:audiomute" }
+String Bravia_Channel "Channel [%s]" { channel="sony:bravia:home:channel" }
+String Bravia_TripletChannel "Triplet Channel [%s]" { channel="sony:bravia:home:tripletchannel" }
+String Bravia_InputSource "Input Source [%s]" { channel="sony:bravia:home:inputsource" }
+String Bravia_Input "Input [%s]" { channel="sony:bravia:home:input" }
+Switch Bravia_PictureMute "Picture Mute [%s]" { channel="sony:bravia:home:picturemute" }
+Switch Bravia_TogglePictureMute "Toggle Picture Mute [%s]" { channel="sony:bravia:home:togglepicturemute", autoupdate="false"  }
+Switch Bravia_Pip "PIP [%s]" { channel="sony:bravia:home:pip" }
+Switch Bravia_TogglePip "Toggle PIP [%s]" { channel="sony:bravia:home:togglepip", autoupdate="false" }
+Switch Bravia_TogglePipPosition "Toggle PIP Position [%s]" { channel="sony:bravia:home:togglepipposition", autoupdate="false" }
+String Bravia_BroadcastAddress "Broadcast [%s]" { channel="sony:bravia:home:broadcastaddress" }
+String Bravia_MacAddress "MAC [%s]" { channel="sony:bravia:home:macaddress" }
+```
+
+demo.sitemap:
+
+```
+sitemap demo label="Main Menu"
+{
+    Frame label="Sony Bravia" {
+        Selection item=Bravia_IR mappings=[18="1",19="2",20="3",21="4",22="5",23="6",24="7",25="8",26="9",27="0",33="Channel Up",44="Channel Down",98="Unknown",101="Unknown2"]
+        Switch item=Bravia_Power
+        Slider item=Bravia_Volume
+        Switch item=Bravia_AudioMute
+        Selection item=Bravia_Channel mappings=[4.1="4.1", 5.1="5.1", 5.2="5.2", 50.1="50.1", 50.2="50.2"]
+        Text item=Bravia_TripletChannel
+        Selection item=Bravia_InputSource mappings=[atsct="ATSCT", dvbt="DVBT", dvbc="DVBC", dvbs="DVBS", isdbt="ISDBT", isdbbs="ISDBBS", isdbcs="ISDBCS", antenna="Antenna", cable="Cable", isdbgt="ISDBGT"]
+        Selection item=Bravia_Input mappings=[TV="TV", HDMI1="HDMI1", HDMI2="HDMI2"]
+        Switch item=Bravia_PictureMute
+        Switch item=Bravia_TogglePictureMute mappings=[ON="Toggle"]
+        Switch item=Bravia_Pip
+        Switch item=Bravia_TogglePip mappings=[ON="Toggle"]
+        Switch item=Bravia_TogglePipPosition mappings=[ON="Toggle"]
+        Text item=Bravia_BroadcastAddress
+        Text item=Bravia_MacAddress
+    }
+}
+```
+
+## IRCC Codes
+The following is a list of codes documented - there may be more or some may have changed since the document has been published.
+
+0=Power Off
```

Comment:
better put that into a table, list or code section - as it is now, it is rendered as a text without line breaks.